tutorials/surface_vehicles.md

```diff
@@ -0,0 +1,378 @@
+with your favorite editor and look at the `Surface` plugin:
+
+```xml
+<!-- Uncomment to produce buoyancy on the left hull -->
+<plugin
+  filename="libSurface.so"
+  name="maritime::Surface">
+  <link_name>base_link</link_name>
+  <hull_length>4.9</hull_length>
+  <hull_radius>0.213</hull_radius>
+  <fluid_level>0</fluid_level>
+  <points>
+    <point>0.6 1.03 0</point>
+    <point>-1.4 1.03 0</point>
+  </points>
+  <wavefield>
+    <topic>/gazebo/wavefield/parameters</topic>
+  </wavefield>
+</plugin>
+```
+
+Before we go into the weeds of the plugin, you can guess from the comment that
+this plugin simulates the buoyancy generated by the left pontoon. A very similar
+plugin is attached to the right pontoon.
+
+The `Surface` plugin simulates the buoyancy of an object at the surface of
+a fluid. This system/plugin must be attached to a model and the system applies
+buoyancy to a collection of points around a given link.
+
+This plugin models the vehicle's buoyancy assuming a single hull with a
+cylindrical shape. You can instantiate multiple plugins when your vehicle has
+multiple pontoons.
+
+This plugin also supports waves. If you provide a wavefield topic via SDF, the
+plugin will receive the wavefield parameters and account for the delta Z that
+the waves generate at each point.
+
+Here are the detailed parameters:
+
+* `<link_name>` is the name of the link used to apply forces.
+* [Optional] `<hull_length>` is the length of the vessel [m].
+* [Optional] `<hull_radius>` is the radius of the vessel's hull [m].
+* [Optional] `<fluid_level>` is the depth at which the fluid should be in the
+             vehicle
+* [Optional] `<fluid_density>` is the density of the fluid.
+* [Optional] `<points>` contains a collection of points where the forces
+             generated by this plugin will be applied. See the format of each
+             point next:
+* [Optional] `<points><point>` Relative position of the point relative to
+             `link_name`.
+* [Optional] `<wavefield>`: The wavefield parameters.
+             See `~/gazebo_maritime_ws/src/gazebo_maritime/src/Wavefield.hh`.
+
+Go ahead and uncomment the two blocks in
+`~/gazebo_maritime_ws/src/gazebo_maritime/models/wam-v/model.sdf` to
+instantiate the `Surface` plugins and see the effect on your WAM-V:
+
+```bash
+colcon build --merge-install
+gz sim -r src/gazebo_maritime/worlds/sydney_regatta.sdf
+```
+
+Now your vehicle is floating in the water!
+
+### Hydrodynamics
+
+I know, I know, it looks like a dancing catamaran. The reason for that behavior
+is because we don't have any drag yet to oppose the forces of the `Surface`
+plugins. We need hydrodynamics!
+
+We will be using Fossen's equations which describe the motion of a craft through
+the water for this. For better understanding of the parameters here, I would
+refer you to [his book](https://www.wiley.com/en-sg/Guidance+and+Control+of+Ocean+Vehicles-p-9780471941132).
```

Comment:
It's listed for $658 XD Can you add a caveat, like "check your library" or "contact the publisher"

Reply:
Added in 292e12d.